pkg/api/norman/customization/aks/handler.go

```diff
@@ -132,7 +132,14 @@ func (h *handler) ServeHTTP(writer http.ResponseWriter, req *http.Request) {
 		}
 		writer.Write(serialized)
 	case "aksVMSizes":
-		if serialized, errCode, err = listVMSizes(req.Context(), capa); err != nil {
+		if serialized, errCode, err = listVMSizesV1(req.Context(), capa); err != nil {
+			logrus.Errorf("[aks-handler] error getting VM sizes: %v", err)
+			handleErr(writer, errCode, err)
+			return
+		}
+		writer.Write(serialized)
+	case "aksVMSizesV2":
+		if serialized, errCode, err = listVMSizesV2(req.Context(), capa); err != nil {
 			logrus.Errorf("[aks-handler] error getting VM sizes: %v", err)
```

Comment:
`aksVMSizes` and `aksVMSizesV2` produces the exact same error log message. Ideally, we want to be able to distinguish between these 2 situations, so suggesting to tweak new log message slightly:
```suggestion
			logrus.Errorf("[aks-handler] error getting VM sizes (v2): %v", err)
```